src/main/java/net/glowstone/entity/GlowPlayer.java

```diff
@@ -195,7 +196,7 @@
  * @author Graham Edgecombe
  */
 @DelegateDeserialization(GlowOfflinePlayer.class)
-public class GlowPlayer extends GlowHumanEntity implements Player {
+public class GlowPlayer extends GlowHumanEntity implements Player, AnimalTamer {
```

Comment:
I think this is redundant because `GlowPlayer` extends `GlowHumanEntity`, which implements `HumanEntity`, which extends `AnimalTamer`.